model-optimizer/mo/utils/cli_parser_test.py

```diff
@@ -355,6 +356,20 @@ def test_mean_do_not_match_input(self):
         mean_values = parse_tuple_pairs("input_not_present(255),input2(255)")
         self.assertRaises(Error, get_mean_scale_dictionary, mean_values, scale_values, "input1,input2")
 
+    def test_values_match_input_name(self):
+        # before fix cli_parser was confused when input had a substring that matches scale/mean values
```

Comment:
Please, refactor the comment. It is not clear what "fix" you are referring to.

Reply:
Changed this comment to a more meaningful